.github/workflows/images-ghcr.yaml

```diff
@@ -0,0 +1,172 @@
+name: Build and Publish Retina Container Images [GHCR]
+
+on:
+  push:
+    branches: [ main ]
+    tags: [ "v*" ]
+  pull_request:
+    branches: [ main ]
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:
+  retina-images:
+    name: Build Agent Images
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        platform: ["linux"]
+        arch: ["amd64", "arm64"]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - uses: actions/setup-go@v4
+        with:
+          go-version: ">=1.21.0"
+      - run: go version
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Log in to registry
+        run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u $ --password-stdin
+
+      - name: Build/Push Images
+        shell: bash
+        run: |
+          set -euo pipefail
+          echo "TAG=$(make version)" >> $GITHUB_ENV
+          make retina-image \
+            IMAGE_REGISTRY=ghcr.io/microsoft/retina \
+            PLATFORM=${{ matrix.platform }}/${{ matrix.arch }}
+
+  retina-win-images:
+    name: Build Agent Images
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        platform: ["windows"]
+        arch: ["amd64"]
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - uses: actions/setup-go@v4
+        with:
+          go-version: ">=1.21.0"
+      - run: go version
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+        
+      - name: Log in to registry
+        run: echo "${{ secrets.GITHUB_TOKEN }}" | docker login ghcr.io -u $ --password-stdin
+
+      - name: Build/Push Images
+        shell: bash
+        run: |
+          set -euo pipefail
+          echo "TAG=$(make version)" >> $GITHUB_ENV
+          make retina-image-win \
+            IMAGE_REGISTRY=ghcr.io/microsoft/retina \
```

Comment:
Can we use some bash command to figure this out dynamically?
Something like 
```
IMAGE_REGISTRY=ghrc.io/$(git remote get-url origin | sed -e 's/git@github.com://g' -e 's/\.git$//')
```

In the documentation we can recommend you fork our repository and this command would automatically set the repository to `<username>/retina`.

Reply:
added, and also updated the GHA to inject the org/repo from context so that forks will upload to _their_ GHP automatically